.github/workflows/playground_examples_ci_reusable.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: '3.8'
-      - uses: actions/setup-java@v3.7.0
```

Comment:
this fixes the problem that arised today
https://github.com/apache/beam/actions/runs/3628990081/jobs/6120678926